projects/packages/search/composer.json

```diff
@@ -14,6 +14,10 @@
 		]
 	},
 	"scripts": {
+		"build": [
```

Comment:
Note that this will not be called by the GitHub Actions build step. For that to happen, you'd need to implement `build-development` and/or `build-production`.

I recommend implementing both in this case, with NODE_ENV set properly so webpack minifies in the production build but not in the dev build.

Reply:
I think you misunderstood what I was saying when you did a5df156e. The Actions builder needs scripts `build-development` and `build-production` here in composer.json.

Reply:
that's definitely my bad. addressed.